LibVLCSharp.WinForms/LibVLCSharp.WinForms.csproj

```diff
@@ -1,4 +1,4 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
```

Comment:
Let's use `MSBuild.Sdk.Extras` here :)

Reply:
Not sure about that. MSBuild.Sdk.Extras was great in setting up xamarin/WPF project with sdk-style csproj with VS2017. Now that VS 2019 and net core 3 are out, the Windows desktop sdk does that job for VS2019, and I guess dotnet CLI too.
I tried MSBuild.Sdk.Extras 2.x not too long ago, and I didn't manage to make it work with VS 2017 anyway. I'm in favor of migrating to VS 2019.

Reply:
If we can keep support for VS2017 without effort, I say we should.

And referencing `MSBuild.Sdk.Extras` here makes it follow the same convention than the WPF package (i.e. `Microsoft.NET.Sdk.WindowsDesktop` is implicitly referenced AFAIK).

Reply:
Yes, but I fear about the "without effort" part. I'm not sure VS2017 is still officially supported by MSBuild.Sdk.Extras...